app/src/main/res/raw/infinite_level

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please make this file have ".json" suffix and the other one which was renamed.

Reply:
Done